solidity/contracts/fully-backed/FullyBackedECDSAKeepFactory.sol

```diff
@@ -68,14 +68,20 @@ contract FullyBackedECDSAKeepFactory is
     // in `FullyBackedBonding` contract. Minimum delegation deposit determines
     // a minimum value of ether that should be transferred to the bonding contract
     // by an owner when delegating to an operator.
-    uint256 public constant minimumBond = 20 ether;
+    uint256 public constant defaultMinimumBond = 20 ether;
 
     // Signer candidates in bonded sortition pool are weighted by their eligible
     // stake divided by a constant divisor. The divisor is set to 1 ETH so that
     // all ETHs in available unbonded value matter when calculating operator's
     // eligible weight for signer selection.
     uint256 public constant bondWeightDivisor = 1 ether;
 
+    // List of applications that got sortition pool created.
+    address[] applications;
+
+    // Maps an application for which keep was created to each keep address.
+    mapping(address => address) keepApplication;
```

Comment:
I think it's mapping keep to application so instead of 
```
// Maps an application for which keep was created to each keep address.
```

we should have:
```
// Maps a keep to an application for which the keep was created for.
// keep address -> application address
```